src/pages/task/taskTabs/buildBaronAndAnnotations/AnnotationTicketsTable/AnnotationTicketsTable.tsx

```diff
@@ -0,0 +1,243 @@
+          confidenceScore={confidenceScore}
+          loading={loading}
+        />
+      ),
+    },
+    {
+      render: ({
+        issueKey,
+        url,
+        confidenceScore,
+      }: AnnotationTicket): JSX.Element => (
+        <ButtonContainer>
+          {ConditionalWrapper({
+            condition: userCanModify,
+            wrapper: (children: JSX.Element) => (
+              <Popconfirm
+                icon={null}
+                placement="topRight"
+                title={`Do you want to move this ${issueString} to ${
+                  isIssue ? "suspected issues" : "issues"
+                }?`}
+                onConfirm={() => {
+                  handleMove({ url, issueKey, confidenceScore });
+                }}
+                okText="Yes"
+                cancelText="Cancel"
+              >
+                {children}
+              </Popconfirm>
+            ),
+            altWrapper: (children: JSX.Element) => (
+              <Tooltip trigger={children}>
+                You are not authorized to edit failure details
+              </Tooltip>
+            ),
+            children: (
+              <Button
+                size={Size.Small}
+                data-cy={`move-btn-${issueKey}`}
+                disabled={!userCanModify}
+                leftGlyph={<Icon glyph={isIssue ? "ArrowDown" : "ArrowUp"} />}
+              >
+                Move To {isIssue ? "Suspected Issues" : "Issues"}
+              </Button>
+            ),
+          })}
+          {ConditionalWrapper({
+            condition: userCanModify,
+            wrapper: (children: JSX.Element) => (
+              <Popconfirm
+                icon={null}
+                placement="topRight"
+                title={`Do you want to delete this ${issueString}?`}
+                onConfirm={() => {
+                  handleRemove(url, issueKey);
+                }}
+                okText="Yes"
+                cancelText="Cancel"
+              >
+                {children}
+              </Popconfirm>
+            ),
+            altWrapper: (children: JSX.Element) => (
+              <Tooltip trigger={children}>
+                You are not authorized to edit failure details
+              </Tooltip>
+            ),
+            children: (
+              <Button
+                size="small"
+                data-cy={`${issueKey}-delete-btn`}
+                leftGlyph={<Icon glyph="Trash" />}
+                disabled={!userCanModify}
+              />
+            ),
+          })}
+        </ButtonContainer>
+      ),
+    },
+  ];
+
+  const [removeAnnotation] = useMutation<
+    RemoveAnnotationIssueMutation,
+    RemoveAnnotationIssueMutationVariables
+  >(REMOVE_ANNOTATION, {
+    onCompleted: () => {
+      dispatchToast.success(`Successfully removed ${issueString}`);
+    },
+    onError(error) {
+      dispatchToast.error(
+        `There was an error removing the ${issueString}: ${error.message}`
```

Comment:
does it make sense to remove the error message part since we log those details

Reply:
I left it to make it easier for users to triage errors on their own. If the error is something like "Malformed jira url the user can use that to take action without reaching out to us"